src/components/TradeWidget/index.tsx

```diff
@@ -543,8 +558,7 @@ const TradeWidget: React.FC = () => {
   const defaultValidFrom = trade.validFrom || validFromParam
   const defaultValidUntil = trade.validUntil || validUntilParam
 
-  const [priceShown, setPriceShown] = useState<'INVERSE' | 'DIRECT'>('INVERSE')
-
+  const [priceShown, setPriceShown] = useState<'INVERSE' | 'DIRECT'>('DIRECT')
```

Comment:
ubernit: and also not from this PR, but can we just give the type a name (create the type)
i.e.
```ts
type PriceRepresentation =  'INVERSE' | 'DIRECT'
```

Reply:
given that's a easy change, could you consider doing this?